pages/search.vue

```diff
@@ -0,0 +1,211 @@
+                    @click="getSearchResults"
+                  />
+                </form>
+              </div>
+            </div>
+          </div>
+        </section>
+      </div>
+      <section>
+        <div class="content">
+          <!-- search results article river -->
+          <!-- <div v-for="(article, index) in articles" :key="index" class="mb-6">
+          {{ article }}
+        </div> -->
+          <template v-if="articles">
+            <div class="grid gutter-x-xl">
+              <div class="col-1 hidden xl:block"></div>
+              <div class="col">
+                <div
+                  v-for="article in articles.slice(0, articlesToShow)"
+                  :key="article.uuid"
+                >
+                  <v-card
+                    class="mod-horizontal mb-3 lg:mb-5 tag-small"
+                    :image="useImageUrl(article.listingImage)"
+                    :width="318"
+                    :height="212"
+                    :sizes="[1]"
+                    :quality="80"
+                    :title="article.listingTitle || article.title"
+                    :titleLink="article.link"
+                    :maxWidth="article.image?.width"
+                    :maxHeight="article.image?.height"
+                    :tags="[
+                      {
+                        name: article.section.name,
+                        slug: `/${article.section.slug}`,
+                      },
+                    ]"
+                  >
+                    <p class="desc">
+                      {{ article.description }}
+                    </p>
+                    <v-card-metadata :article="article" />
+                  </v-card>
+                  <hr class="mb-5" />
+                </div>
+                <Button
+                  v-if="articlesToShow < articles.length"
+                  class="p-button-rounded"
+                  label="Load More"
+                  @click="articlesToShow += 6"
+                >
+                </Button>
+              </div>
+              <div class="col-fixed hidden xl:block mx-auto">
+                <img
+                  class="mb-4 xl:mb-7"
+                  src="https://fakeimg.pl/300x600/?text=AD Here"
+                  style="max-width: 100%"
+                  width="300"
+                  height="600"
+                  alt="advertisement"
+                />
+              </div>
+            </div>
+          </template>
+          <!-- newsletter -->
+          <div class="mt-8 mb-5">
+            <hr class="black mb-4" />
+            <newsletter-home @submit="newsletterSubmitEvent" />
+          </div>
+        </div>
+      </section>
+    </div>
+  </div>
+</template>
+
+<style lang="scss">
+.search {
+  background: linear-gradient(
+    180deg,
+    #f3f3e4 0,
+    rgba(255, 255, 255, 0) 720px,
+    rgba(255, 255, 255, 0) 100%
+  );
+  form {
+    max-width: 894px;
+    width: 100%;
+    position: relative;
+  }
+  .search-page-input {
```

Comment:
after changing the button to right: 0.5rem, add padding-right: 50px; so the text does not overlap the button